api/users.go

```diff
@@ -37,6 +40,20 @@ func (c *Client) GetUsers(accessToken string, reqRealmName, targetRealmName stri
 	return resp, err
 }
 
+// GetLockedUsers returns a list of locked user IDs
+func (c *Client) GetLockedUsers(accessToken, reqRealmName, targetRealmName string) ([]string, error) {
```

Comment:
Why those methods are not part of users ? (i.e. /realms/:realm/users/locked-users)